Ref #8285. Updates on Zephyr emulator (#154)
micro-ROS/micro-ROS.github.io: _docs/tutorials/advanced/zephyr_emulator/index.md

This tutorial requires no hardware beyond a Linux host computer.

## Building a Zephyr emulator application

> Once the micro-ROS build system is ready, let's create a new Zephyr firmware for host platform:

Reviewer: ```suggestion
Once the micro-ROS build system is ready, let's create a new Zephyr firmware for the host platform:
```

```bash
# Create firmware step
ros2 run micro_ros_setup create_firmware_ws.sh zephyr host
```

> micro-ROS apps for Zephyr emulator are located at `firmware/zephyr_apps/apps`. In order to create a new application, create a new folder containing two files: the app code (inside a `src` folder) and the RMW configuration. You will also need some other Zephyr related files: a `CMakeLists.txt` to define the building process and a `prj.conf` where Zephyr is configured. There is a sample proyect [here](https://github.com/micro-ROS/zephyr_apps/tree/dashing/apps/host_ping_pong), for now, it is ok to copy them.

Reviewer: ```suggestion
micro-ROS apps for the Zephyr emulator are located at `firmware/zephyr_apps/apps`. In order to create a new application, create a new folder containing two files: the app code (inside a `src` folder) and the RMW configuration. You will also need some other Zephyr related files: a `CMakeLists.txt` to define the building process and a `prj.conf` where Zephyr is configured. You can find a sample project [here](https://github.com/micro-ROS/zephyr_apps/tree/dashing/apps/host_ping_pong). For this tutorial, it is ok to copy them.
```

Reviewer: Maybe I liked better when all this was shown rather than explained. I mean, just the commands, and then references to where to find the files content to copy.
I mean, I would use the same text as the old tutorial, with the pushd and popd commands and everything, and only change the way to refer the user to the files content.

Author: Done

Reviewer: Nice. Can't you make those links cliackable?

Author: Done

Make sure your app's `CMakeLists.txt` is compatible with `native_posix`:

```

> ...

Reviewer: I guess it's obvious for a normal user, but is it a shell command or a line to add into the CMakeLists.txt?
I guess the second, but maybe it can be specified.
